modules/nf-core/bbmap/bbmerge/main.nf

```diff
@@ -0,0 +1,44 @@
+process BBMAP_MERGE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bbmap:39.01--h5c4e2a8_0':
+        'biocontainers/bbmap:39.01--h5c4e2a8_0' }"
+
+    input:
+    tuple val(meta), path(fastq)
+    
+    output:
+    tuple val(meta), path("*_merged.fastq"), emit: merged_fastq
+    tuple val(meta), path("*_unmerged_R1.fastq"), emit: unmerged_fastq_1
```

Comment:
Should these be optional?
Can you line up the commas in the emit statements please.

Reply:
Yep they can be optional cheers! I've made the other suggested changes too. Thanks for the review.